nss-day-cohort-29/reactive-nutshell-copperden
# Ab events

# Description
Sorted events by nearest date to furthest date regardless of date format. Added "Add Event" functionality and "Delete Event" functionality. Alert should post for confirmation of delete. If forms are not filled for the Add Event, an alert is posted warning that the field is empty.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout ab-events`
3. Run server
4. Verify that add button and delete buttons work properly. Test that no input in the "Add Event" form produces an alert relevant to the incomplete form. Verify that events are listed in order from nearest date to furthest date. (Format of the date should not matter)

# Checklist:
